pkg/agent/controller/traceflow/packetin.go

```diff
@@ -134,7 +136,6 @@ func (c *Controller) parsePacketIn(pktIn *ofctrl.PacketIn) (*crdv1alpha1.Tracefl
 		ob := new(crdv1alpha1.Observation)
 		ob.Component = crdv1alpha1.ComponentForwarding
 		ob.Action = crdv1alpha1.ActionReceived
-		ob.ComponentInfo = openflow.GetFlowTableName(openflow.ClassifierTable)
```

Comment:
@gran-vmv : I feel it is not really useful to add ClassifierTable to the result, as it does not provide any meaningful information. Let me know if you do not agree.

Reply:
I'm OK for this change. ob.Component and ob.Action can uniquely define this case.